regression/goto-analyzer/constant_propagation_08/test-vsd.desc

```diff
@@ -3,7 +3,7 @@ main.c
 --variable-sensitivity --vsd-arrays every-element --simplify out.gb
 ^EXIT=0$
 ^SIGNAL=0$
-^Simplified:  assert: 1, assume: 0, goto: 1, assigns: 3, function calls: 0$
-^Unmodified:  assert: 0, assume: 0, goto: 2, assigns: 7, function calls: 2$
+^Simplified:  assert: 1, assume: 0, goto: 1, assigns: [12], function calls: 0$
```

Comment:
Why the brackets?

Reply:
Ah, wait, is this platform dependent results?  If so this feels kinda bad.

Reply:
Fixed in #7912.